helpers.rb

```diff
@@ -0,0 +1,35 @@
+module Helpers
+  def response(name)
+    sleep 1
+    puts "\n#{name} created successfully!!!"
+    sleep 1
+    continue?
+  end
+
+  def continue?
+    print "\nDo you wish to continue? [Y/N]: "
+    answer = gets.chomp
+
+    if answer.downcase == 'y' || answer.downcase == 'yes' || answer == ''
+      clear
+      prompt_user
+    else
+      exit
+    end
+  end
```

Comment:
- Kindly look into the way you connect your `continue` method and make sure that it works. Currently, when I choose yes to continue your UI takes me to a blank screen. I presume it should be taking me back to the main menu or exit if I choose NO.

![image](https://user-images.githubusercontent.com/38088818/174103013-4269ff7b-573c-40b9-b5cb-89b98af3ab01.png)